rep-2000.rst

```diff
@@ -1179,6 +1179,120 @@ Package manager use for dependencies:
+|    arm64     | Tier 1 [d][a][s] |               |                  |             |   Tier 3 [s]    |   Tier 3 [s]   |
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+|    arm32     |    Tier 3 [s]    |               |                  |             |   Tier 3 [s]    |   Tier 3 [s]   |
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Distribution-specific (Debian, RPM, etc.) packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Jazzy ROS 2 repos file [14]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| Middleware Library       | Middleware Provider    | Support Level | Platforms                  | Architectures                  |
++==========================+========================+===============+============================+================================+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS      | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS    | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_connextdds           | RTI Connext            | Tier 1        | Ubuntu, Windows, and macOS | All Architectures except arm64 |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS      | Tier 2        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_gurumdds_cpp         | GurumNetworks GurumDDS | Tier 3        | Ubuntu and Windows         | All Architectures except arm32 |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.8
```

Comment:
Windows is holding us back right now. Are we planning to bump this? 3.8 is already EOL and 3.9 will go EOL in October.
```suggestion
- Python 3.9
```

Reply:
This depends on the work to switch to Conda.  In particular, assuming we get that work done, we'll switch Windows to Python 3.12 (to match Ubuntu 24.04).  At that point, RHEL-9 will be the oldest one.  So I'd say let's make this 3.9.